File: Day2/src/main/java/com/example/carnextdoor/model/Entitiec/Owner.java
package com.example.carnextdoor.model.Entitiec;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

@Entity
@Table
public class Owner {
@Id
    private int owner_id;
    private String owner_firstName;
    private String owner_lastName;
    private String owner_address;
    private int owner_phone;
    private int owner_age;

    public Owner(int owner_id, String owner_firstName, String owner_lastName, String owner_address, int owner_phone, int owner_age) {
        this.owner_id = owner_id;
        this.owner_firstName = owner_firstName;
        this.owner_lastName = owner_lastName;
        this.owner_address = owner_address;
        this.owner_phone = owner_phone;
        this.owner_age = owner_age;
    }

    public Owner() {
    }

    public int getOwner_id() {
        return owner_id;
    }

    public void setOwner_id(int owner_id) {
        this.owner_id = owner_id;
    }

    public String getOwner_firstName() {
        return owner_firstName;
    }

    public void setOwner_firstName(String owner_firstName) {
        this.owner_firstName = owner_firstName;
    }

    public String getOwner_lastName() {
        return owner_lastName;
    }

    public void setOwner_lastName(String owner_lastName) {
        this.owner_lastName = owner_lastName;
    }

    public String getOwner_address() {
        return owner_address;
    }

    public void setOwner_address(String owner_address) {
        this.owner_address = owner_address;
    }

    public int getOwner_phone() {
        return owner_phone;
    }

    public void setOwner_phone(int owner_phone) {
        this.owner_phone = owner_phone;
    }

    public int getOwner_age() {
        return owner_age;
    }

    public void setOwner_age(int owner_age) {
        this.owner_age = owner_age;
    }

    @Override
    public String toString() {
        return "Owner{" +
                "owner_id=" + owner_id +
                ", owner_firstName='" + owner_firstName + '\'' +
                ", owner_lastName='" + owner_lastName + '\'' +
                ", owner_address='" + owner_address + '\'' +
                ", owner_phone=" + owner_phone +
                ", owner_age=" + owner_age +
                '}';
    }
}

File: Front End/router tutorial/router tutorial/src/routes/Owner.js
import React from "react";
import react, {useState } from 'react';
import axios from "axios";

export default function Owner() {
    let [owner_id, setowner_id] = useState("")
    let [firstName, setOwner_firstNamE] = useState("")
    let [lastName, setOwner_lastName] = useState("")
    let [Age, setOwner_age] = useState("")
    let [Email, setOwner_email] = useState("")
    let [Address, setOwner_address] = useState("")
    let [phone, setOwner_phone] = useState("")


    function handleOwner_id(event) {
        setowner_id((owner_id = event.target.value));
    }
    function handleOwner_firstName(event) {
        setOwner_firstNamE((firstName = event.target.value));
    }
    function handleOwner_lastName(event) {
        setOwner_lastName((lastName = event.target.value));
    }
    function handleOwner_age(event) {
        setOwner_age((Age = event.target.value));
    }
    function handleOwner_email(event) {
        setOwner_email((Email = event.target.value));
    }
    function handleOwner_address(event) {
        setOwner_address((Address = event.target.value));
    }
    function handlEOwner_phone(event) {
        setOwner_phone((phone = event.target.value));
    }


    function rgisterHandel() {
        let NewOwner = { owner_id: owner_id, owner_firstName: firstName, owner_lastName: lastName, owner_age: Age, owner_email: Email, owner_address: Address, owner_phone: phone }

        console.log("insid function");
        console.log(NewOwner)
        axios({
            method: 'post',
            url: 'owner/add',
            data: NewOwner,
        });
    }


    return (

        <div>
            <h1 > Register NewOwner </h1>
            <br />
            <label> OwnerId: </label>
            <input type="text" name="OwnerId"
                placeholder="OwnerId"
                onChange={handleOwner_id}
            />
            <br />  <br />
            <label> Firstname: </label>
            <input type="text" name="firstName"
                placeholder="FirstName"
                onChange={handleOwner_firstName}
            />
            <br />  <br />

            <label className="last"> Lastname: </label>
            <input type="text" name="lastName"
                placeholder="LastName"
                onChange={handleOwner_lastName}
            />
            <br />  <br />

            <label> Age: </label>
            <input type="text" name="age"
                placeholder="Age" onChange={handleOwner_age}
            />
            <br />  <br />
            <label> Email: </label>
            <input type="text" name="email"
                placeholder="email" onChange={handleOwner_email}
            />
            <br />  <br />
            <label> Address: </label>
            <input type="text" name="Address"
                placeholder="Address" onChange={handleOwner_address}
            />
            <br />  <br />
            <label> Phone: </label>
            <input type="text" name="phone"
                placeholder="phone" onChange={handlEOwner_phone}
            />
            <br />  <br />
            <button onClick={rgisterHandel}>Submit</button>

        </div>

    );
}